test/hotspot/jtreg/compiler/c2/irTests/blackhole/BlackholeStoreStoreEATest.java

```diff
@@ -39,6 +39,7 @@ public class BlackholeStoreStoreEATest {
 
     public static void main(String[] args) {
         TestFramework.runWithFlags(
+            "-XX:+UseTLAB",
```

Comment:
Just as a side note: As this example shows, treating `UseTLAB` as having no effect on the IR (i.e. on the IR framework whitelist) is incorrect. We should think about removing it from the whitelist at some point. But fixing this test like that is perfectly fine.